build-tools/packages/build-cli/src/lib/bump.ts

```diff
@@ -172,41 +172,50 @@ export async function bumpReleaseGroup(
 		: bumpType;
 
 	let name: string;
-	const cmds: [string, string[]][] = [];
-	let workingDir: string;
+	const cmds: [string, string[], execa.Options | undefined][] = [];
+	let options: execa.Options | undefined;
 
 	// Run npm version in each package to set its version in package.json. Also regenerates packageVersion.ts if needed.
 	if (releaseGroupOrPackage instanceof MonoRepo) {
-		workingDir = context.gitRepo.resolvedRoot;
 		name = releaseGroupOrPackage.kind;
+		options = {
+			cwd: releaseGroupOrPackage.repoPath,
+			stdio: "inherit",
+			shell: true,
+		};
 		cmds.push(
-			[`flub`, [`exec`, "-g", name, "--", `"npm version ${translatedVersion.version}"`]],
-			[`npm`, ["run", "build:genver"]],
+			[
+				`flub`,
+				[`exec`, "-g", name, "--", `"npm version ${translatedVersion.version}"`],
+				options,
+			],
+			["pnpm", ["-r", "run", "build:genver"], options],
 		);
 	} else {
-		workingDir = releaseGroupOrPackage.directory;
 		name = releaseGroupOrPackage.name;
-		cmds.push([`npm`, ["version", translatedVersion.version]]);
+		options = {
+			cwd: releaseGroupOrPackage.directory,
+			stdio: "inherit",
+			shell: true,
+		};
+		cmds.push([`npm`, ["version", translatedVersion.version], options]);
 		if (releaseGroupOrPackage.getScript("build:genver") !== undefined) {
-			cmds.push([`npm`, ["run", "build:genver"]]);
+			cmds.push([`pnpm`, ["run", "build:genver"], options]);
```

Comment:
Is it necessary to change this to `pnpm`? For commands isn't that just an alias for `npm` anyways? Just thinking in terms of overall flexibility - is the tooling locked in to `pnpm` instead of standard `npm`?

Reply:
```suggestion
			cmds.push([`npm`, ["run", "build:genver"], options]);
```

No, I don't think this change is needed. Probably leftover from some debugging or something.